test/domUpdates-test.js

```diff
@@ -3,49 +3,44 @@ const chai = require('chai');
 const expect = chai.expect;
 const spies = require("chai-spies");
 import User from '../src/User';
-import DomUpdates from '../src/domUpdates';
+import domUpdates from '../src/domUpdates';
```

Comment:
Let's get a paired session going so we all can have a go at when and how to implement spies